### PR TITLE
Add modify signature in invoke contract dialog

### LINKED ISSUE #305: Creating transactions with same hash and no signature
Some exchanges and some developers have reported this issue: when they invoke the same method in the same smart contract multiple times and neo-gui create invocation transactions under 10 gas free limit, these transactions will have the same hash and have no signatures, which causes these txs are never confirmed. 

A temporary solution I provided to them is adding the timestamp and the first address in neo-gui in the transaction attributes. 

We need a permanent fix for this issue. Any suggestion is welcome.

### PR DESCRIPTION
**Pull request by Lichen9618**
Fixes #305

### CODE REVIEW

**Comment by shargon** [comment]
It should not be general for the three options?

image

**Comment by Lichen9618** [comment]
Signature function works for three tab now. (It has to be clicked in fucntion tab)
But if I change the design to
image
It looks wired.
Do you have any suggestion?
 @shargon

**Comment by shargon** [comment]
I think that should be like your image or repeated in every tab. I prefer like you show us

**Comment by Lichen9618** [comment]
Get it, I will change the layout soon.

File: neo-gui/UI/InvokeContractDialog.Designer.cs [hunk 1]
```diff
@@ -303,16 +306,36 @@ private void InitializeComponent()
...
+            resources.ApplyResources(this.openFileDialog2, "openFileDialog2");
+            // 
+            // label_Signature
+            // 
+            resources.ApplyResources(this.label_Signature, "label_Signature");
+            this.label_Signature.Name = "label_Signature";
+            //
+            // comboBoxSignature
+            // 
+            this.comboBoxSignature.FormattingEnabled = true;
```
Comment (shargon) [inline neo-gui/UI/InvokeContractDialog.Designer.cs:319]:
```suggestion
            this.comboBoxSignature.FormattingEnabled = true;
            this.comboBoxSignature.DropDownStyle = System.Windows.Forms.ComboBoxStyle.DropDownList;
```

File: neo-gui/UI/InvokeContractDialog.cs [hunk 2]
```diff
@@ -203,5 +241,30 @@ private void comboBox1_SelectedIndexChanged(object sender, EventArgs e)
             UpdateParameters();
             UpdateScript();
         }
+
+        private void Button9_Click(object sender, EventArgs e)
+        {
+            if (comboBoxSignature.SelectedItem.ToString() == "")
+            {
+                MessageBox.Show("Please choose address");
```
Comment (shargon) [inline neo-gui/UI/InvokeContractDialog.cs]:
```suggestion
                MessageBox.Show("Please choose the address");
```

**Review by chenzhitong — APPROVED** [review]

**Pull request merged**